### ISSUE TITLE
Problem with the color picker tool

### ISSUE THREAD

**Issue by Wiering** [Orama-Interactive/Pixelorama#115]
This is a a personal preference, but I really like to be able to draw with the left mouse button and pick up a color with the right button (as most pixel editors allow you to do).

I like how you can choose tools for each button, but if I choose to draw with left and pick up with right, it only picks the background color and I draw with the foreground color. That is very annoying.

A solution would be if you allow the user to click the color picker a second time, it turns to the other color (from red to blue in this case) indicating that you pick up the foreground color instead of the background color.

Alternatively, you could also allow to switch to a mode where you only have one color instead of a foreground and background color.

**Comment by OverloadedOrama** [comment]
Hey there. We could have the option to pick for the left or the right tool in the tool options, when the user has selected the color picker tool, like this 
![image](https://user-images.githubusercontent.com/35376950/72171759-4adc6180-33dc-11ea-90d2-fdd0b360bd72.png)

Do you like this idea?

**Comment by Wiering** [comment]
Yes, that would also be fine.

**Comment by OverloadedOrama** [comment]
Will implement this on the next commit, thank you! :)

**Comment by hsandt** [comment]
Oh, I had the same question and just learned about this option now!

... but I found this issue by accident while looking for another, unrelated Color Picker issue.

@Wiering Consider renaming the issue title to something more specific to help other looking for this feature finding out it was implemented, and to indicate to users looking for something else that this is not the issue they are looking for.

For instance, the title could be "Feature request: let Color Picker assign left/right color independently from whether it's left/right tool"
